### ISSUE TITLE
Benchmark:  Conv Net - Accuracy: 92.56%

### ISSUE THREAD

**Issue by umbertogriffo** [zalandoresearch/fashion-mnist#59]
Tried this network topology that can be summarized as follows:

- Convolutional layer with 32 feature maps of size 5×5.
- Pooling layer taking the max over 2*2 patches.
- Convolutional layer with 64 feature maps of size 5×5.
- Pooling layer taking the max over 2*2 patches.
- Convolutional layer with 128 feature maps of size 1×1.
- Pooling layer taking the max over 2*2 patches.
- Flatten layer.
- Fully connected layer with 1024 neurons and rectifier activation.
- Dropout layer with a probability of 50%.
- Fully connected layer with 510 neurons and rectifier activation.
- Dropout layer with a probability of 50%.
- Output layer.

I used Normalization as Preprocessing and 5-fold cross-validation to evaluate the model.
Accuracy scores: [0.92433, 0.92133, 0.923581, 0.92391, 0.92466]
Mean Accuracy: 0.923567
Stdev Accuracy: 0.001175
Final Accuracy: 92.56%

You can find the code [here](https://github.com/umbertogriffo/Fashion-mnist-cnn-keras).

**Comment by hanxiao** [comment]
Thanks for your contribution, two remarks:

1. you may want to evaluate test accuracy on the test set (10,000 examples)
2. Keras now include Fashion-MNIST as a built-in dataset! 😃 So you don't need to download from Kaggle anymore.

**Comment by umbertogriffo** [comment]
You are welcome.

1. After evaluating the model using the 5-fold cross-validation (on 60,000 examples divided into train and test) I ran a new learning from scratch (on 60,000 examples) and then I evaluated test accuracy on the test set (10,000 examples).
2. Nice news! 😃
